client/coral-admin/src/routes/Moderation/containers/Moderation.js

```diff
@@ -208,7 +212,7 @@ const withModQueueQuery = withQuery(gql`
     }) {
       ...CoralAdmin_Moderation_CommentConnection
     }
-    assets: assets {
+    assets: assets(query: {}) {
```

Comment:
This breaks current moderation, because it assumes the whole list of assets. 
The proposed changes would limit the results to only 10 assets.

![image](https://user-images.githubusercontent.com/14221600/27238416-f70755f6-52f6-11e7-8fc2-72252d455886.png)